style.css

```diff
@@ -0,0 +1,89 @@
+body {
+  background-image: url(pics/shapes.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  color: #6070ff;
+  transition: color 1s ease;
+}
+
+.logo:hover {
+  color: darksalmon;
+}
+
+#items {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```

Comment:
Great job on getting your project to resemble the figma design. however, this project requires that you follow the figma design's font, colors and also space between the elements
## currently this is your design
<img width="1595" alt="Screenshot 2022-09-22 at 16 58 00" src="https://user-images.githubusercontent.com/48546992/191767658-89f825b3-1a96-48c2-8645-6917e5a0d333.png">
ts as well as arrangement.

## this is the figma desidn required.
<img width="182" alt="Screenshot 2022-09-22 at 16 50 13" src="https://user-images.githubusercontent.com/48546992/191767998-2dc008b0-81bc-473b-b1f5-d701492f5c94.png">
 

- [ ] Kindly note that its a requirement that the design has the media images below to social media pages.